app/src/main/java/com/mybus/helper/FileLoaderHelper.java

```diff
@@ -71,4 +91,23 @@ private static String getStringFromResource(Object obj, String filename) throws
         is.close();
         return new String(buffer, "UTF-8");
     }
+
+
+
+    /**
+     * Gets a String from specific assets name
+     *
+     * @param context
+     * @param filename
+     * @return
+     * @throws IOException
+     */
+    private static String getStringFromAssets(Context context, String filename) throws IOException {
+        InputStream is = context.getResources().getAssets().open(filename, Context.MODE_WORLD_READABLE);
```

Comment:
I would suggest you to use Commons IO:  

```java
IOUtils.toString(inputStream, StandardCharsets.UTF_8.name());
```

There are more options here: http://www.baeldung.com/convert-input-stream-to-string

But If you want to handle the InputStream directly you should use try-with-resources because InputStream is AutoCloseable.

```java
try(InputStream is = context.getResources().getAssets().open(filename, Context.MODE_WORLD_READABLE)) {
    ...
}
```

And then remove:

```java
is.close();
```